examples/ClimateMachine/README.md

```diff
@@ -0,0 +1,25 @@
+
+
+This example requires ClimateMachine.jl to be installed in the same parent
+directory as EnsembleKalmanProcesses.jl. 
+
+You may install ClimateMachine.jl directly from GitHub:
+
+>> git clone https://github.com/CliMA/ClimateMachine.jl.git
+
+The example makes use of a simple single column model configuration with two tunable 
+parameters to showcase the use of EnsembleKalmanProcesses.jl with ClimateMachine.jl.
+This example is a perfect model experiment, in the sense that the ground truth is 
+generated using the same model and a certain combination of parameters.
+
+To run the example using a SLURM workload manager, simply do:
+
+>> sbatch ekp_calibration.sbatch
+
```

Comment:
Maybe state a little more here, such as
```
This will create a queue on the workload manager running 
1. ekp_init_calibration (creating initial parameters)
Followed by iterating between
2a. ekp_single_cm_run (running the GCM)
2b. ekp_cont_calibration (updating parameters with EKP)
until the desired number of iterations (defined in ekp_calibration.sbatch) are complete.
```

Reply:
RE the truth output here - 
1. Is it something that the user can generate themselves from the true parameters? If so, then we don't require the nc data file upload on this repo, and instead we should have a script to generate the truth? 
2. If it is instead something that the users cannot easily generate then maybe just say here in the readme that we used parameters `(c_1,c_2) = (...)` to generate the true data stored in `truth_output/SBL_ANELASTIC_1D_AtmosLESDefault.nc`, it is the goal of EKP to recover these.

Reply:
Otherwise LGTM - if you confirmed the examples run?